tools/mo/openvino/tools/mo/convert_impl.py

```diff
@@ -938,38 +1007,21 @@ def _convert(cli_parser: argparse.ArgumentParser, framework, args):
         inp_model_is_object = input_model_is_object(args)
         if inp_model_is_object:
             model_framework = check_model_object(args)
-            if model_framework == "pytorch" and not os.environ.get('USE_PYTORCH_FRONTEND'):
-
-                opset_version = None
-                if 'onnx_opset_version' in args and args['onnx_opset_version'] is not None:
-                    opset_version = args['onnx_opset_version']
-
+            if model_framework == "pytorch":
+                use_pt_fe = not args.get("use_legacy_frontend", False)
                 example_inputs = None
                 if 'example_input' in args and args['example_input'] is not None:
                     example_inputs = args['example_input']
+                if not use_pt_fe:
+                    # TO DO: remove this path, when pytorch frontend productization is finished, CVS-103726
+                    if "use_legacy_frontend" in args:
+                        args.pop("use_legacy_frontend")
```

Comment:
Am I correct that we need to do this in order to avoid use of Python ONNX FE? Can you leave a comment here about it?